### PR TITLE
Updating prime contracts

### PR DESCRIPTION
**Pull request by IanWoodard**

### CODE REVIEW

File: prime/src/pages/BorrowActionsPage.tsx [hunk 1]
```diff
@@ -288,6 +288,7 @@ export default function BorrowActionsPage() {
           setMarginAccount(result.marginAccount);
         }
       } catch (error) {
+        console.log(error);
```
Comment (haydenshively) [inline prime/src/pages/BorrowActionsPage.tsx:291]: 😮

File: prime/src/data/BalanceSheet.ts [hunk 2]
```diff
@@ -346,7 +353,16 @@ export function computeLiquidationThresholds(
   const scaler = 10 ** (token0Decimals - token1Decimals);
```
Comment (haydenshively) [inline prime/src/data/BalanceSheet.ts:353]: We adjusted these bounds in Earn. Use the numbers from there?

File: prime/src/components/borrow/ManageAccountTransactionButton.tsx [hunk 3]
```diff
@@ -218,69 +233,47 @@ export function ManageAccountTransactionButton(props: ManageAccountTransactionBu
...
-      calldata,
-      [
-        isRemovingToken0Collateral || isSwappingToken0ForToken1,
-        isRemovingToken1Collateral || isSwappingToken1ForToken0,
-      ],
-    ],
-    overrides: { value: shouldProvideAnte ? ANTE + 1 : undefined },
-    enabled: canConstructTransaction && enabled && !transactionWillFail && !needsApproval[0] && !needsApproval[1],
+    args: [ALOE_II_FRONTEND_MANAGER_ADDRESS[activeChain.id], calldata as `0x${string}`, oracleSeed ?? Q32],
+    overrides: { value: shouldProvideAnte ? ante.recklessAdd(1).toBigNumber() : undefined },
```
Comment (haydenshively) [inline prime/src/components/borrow/ManageAccountTransactionButton.tsx:268]: Don't need to recklessAdd(1) anymore.

File: prime/src/components/borrow/ManageAccountTransactionButton.tsx [hunk 4]
```diff
@@ -218,69 +233,47 @@ export function ManageAccountTransactionButton(props: ManageAccountTransactionBu
...
+    chainId: activeChain.id,
+    enabled: enabled,
+  });
+
+  const ante = useMemo(() => {
+    if (!anteData) return GN.zero(18);
+    return GN.fromBigNumber(anteData[0], 18);
+  }, [anteData]);
+
+  const gnAccountEtherBalance = accountEtherBalance ? GN.fromBigNumber(accountEtherBalance.value, 18) : GN.zero(18);
```
Comment (haydenshively) [inline prime/src/components/borrow/ManageAccountTransactionButton.tsx:250]: No reason to have useMemo here

**Review by haydenshively — APPROVED** [review]

File: prime/src/components/borrow/ManageAccountTransactionButton.tsx (hunk 3 above)
Comment (IanWoodard) [reply prime/src/components/borrow/ManageAccountTransactionButton.tsx:265]: Punting, remind me when this change takes effect : )

**Pull request merged**